src/components/PracticeSection/MultipleChoice.js

```diff
@@ -317,8 +317,8 @@ export default function MultipleChoice() {
           student.add(category + "_correct_ids", currentQuestionId);
           student.increment("total_correct_questions");
           var correct = student.get(category + "_correct_ids");
-          if (correct.length === 7) {
-            if (studentLevel === 3) {
+          if (correct.length === 1) {
```

Comment:
I'm not a 100% sure what this part does, so my question might be weird.
Why are we changing this from (7,3) to (1,2) ?

Reply:
The problem might stem from the hardcoded numerals. 
If we gave them symbolic names and defined as constants the code would be more readable.

Reply:
This is my bad. 
It was a test and I forgot to reset it. The values should still be (7,3)